pkgs/by-name/ma/maturin/package.nix

```
@@ -25,6 +25,12 @@ rustPlatform.buildRustPackage rec {
   useFetchCargoVendor = true;
   cargoHash = "sha256-6aLWkphWLScUz6l0RJj9LmNad6aPxLz2iVxXVOXq7pg=";
 
+  patches = [
+    # Sorts RECORD file in wheel archives to make them deterministic. See: https://github.com/NixOS/nixpkgs/issues/384708
+    # Remove on next bump https://github.com/PyO3/maturin/pull/2550
+    ./wheel-deterministic-record.patch
```

Would be nice to avoid vendoring this when it's been merged upstream

```suggestion
    (fetchpatch {
      name = "wheel-deterministic-record.patch";
      url = "https://github.com/PyO3/maturin/commit/bade37e108514f4288c1dd6457119a257bf95db4.patch";
      hash = "sha256-jcZ/NMHKFYQuOfR+fu5UPykEljUq3l/+ZAx0Tlyu3Zw=";
    })
```

---

ah yes, now that the patch was merged upstream we can do it like this indeed

---

Nice, I just learned this—thank you 🙏